docs/docs-beta/docs/tutorials/category-two/prompt-engineering/prompts.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Prompts
+description: Generating the first prompt
+last_update:
+  author: Dennis Hume
+sidebar_position: 20
+---
+
+We will be working with the API for the National Renewable Energy Laboratory ([NREL](https://www.nrel.gov/)). This API allows us to find nearby alternative fuel stations. To use this API we need to provide our latitude and longitude and some information about our vehicle. Trying to make this user friendly, we might want to put an AI in front to translate free text into the information we need.
+
+Using AI models like Claude, we can get coordinates based on an address as well as parse information out on our vehicle. For example if we told it: "I'm near the The Art Institute of Chicago and driving a Kia EV9, what are my coordinates and my vehicle type?", it will provide an answer that may sound like this:
+
+> Your coordinates will depend on your exact location near The Art Institute of Chicago, but its approximate coordinates are 41.8796° N, 87.6237° W. If you're nearby, your latitude and longitude should be close to these.
+> 
+> Your vehicle type is a Kia EV9, which is a fully electric SUV. Let me know if you need parking suggestions or other assistance nearby!
+
+This is helpful but has a couple of issues. First it requires the user to know what to include for our backend API. We don't want to count on the user to include that information. Also the response we get back will be hard to parse and we cannot guarantee that it will always be in the same format.
```

Comment:
```suggestion
This is helpful, but has a couple of issues. First, it requires the user to know what to include for our backend API. We don't want to count on the user to include that information. Also, the response we get back will be hard to parse, and we cannot guarantee that it will always be in the same format.
```